yt_dlp/extractor/rai.py

```diff
@@ -82,6 +82,8 @@ def fix_cdata(s):
         geoprotection = xpath_text(relinker, './geoprotection', default='N') == 'Y'
 
         ext = determine_ext(media_url)
+        if ext:
+            ext = ext.lower()
```

Comment:
by default, `determine_ext()` will always return a string, so we can simplify this
```suggestion
        ext = determine_ext(media_url).lower()
```